### ISSUE TITLE
About consistency distillation

### ISSUE THREAD

**Issue by my4f175** [thu-ml/Causal-Forcing#13]
Thanks for the excellent work!
I would like to ask what the loss curve during the Consistency Distillation process generally looks like. Would you be willing to share the loss plot? The loss curve I obtained seems a bit weird. Since this stage trains relatively slowly, I’d like to confirm in advance whether there might be an issue with my training.

**Comment by zhuhz22** [comment]
<img width="1782" height="411" alt="Image" src="https://github.com/user-attachments/assets/86de56a0-175b-4b46-b11f-b0bc87cc648c" />
This is the chunkwise setting.

**Comment by zhuhz22** [comment]
First 100 steps:

<img width="1183" height="379" alt="Image" src="https://github.com/user-attachments/assets/84060171-481a-43a1-bb75-8a2df18158ce" />

**Comment by my4f175** [comment]
Thanks for the kind and helpful reply. My problem is resolved.

**Comment by WJohnnyW** [comment]
Hi,

Thank you for the great work!

I am training on my own AV-joint model based on Ovi. After the S1 AR training stage, the AR generation quality is slightly worse than the base model, but still acceptable.

However, after the S2 Causal CD training stage, I found that the inference results are problematic: the generated video is almost static, and sometimes becomes very blurry. When the person is talking, there is corresponding audio, but the volume is very low.

So far, I have trained for roughly 2700 steps × 14 samples. Do you think this is simply because the training is still insufficient, or could there be something wrong with my training setup?
